recipes/gto/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "gto" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gto-{{ version }}.tar.gz
+  sha256: 65a5bcdfaba622ef58b299dc48529006d3476218b7ea4fe84f9eed2a71c604e2
+
+build:
+  entry_points:
+    - gto = gto.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools-scm
+    - fastentrypoints >=0.12
+    - pip
+  run:
+    - python >=3.6
+    - gitpython
+    - typer >=0.4.1
+    - rich
+    - pydantic
+    - ruamel.yaml
+    - semver ==3.*
+    - entrypoints
+    - tabulate >=0.8.10
+    - funcy
+
+test:
+  imports:
+    - gto
+  commands:
+    - pip check
+    - gto --help
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/gto/
+  summary: Version and deploy your models following GitOps principles
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - bgalvao
+    - janjagusch
```

```suggestion
    - bgalvao
    - janjagusch
    - shcheklein
    - mike0sv
    - aguschin
```

Hi! Please add some folks from iterative.ai :)

---

sure :)